src/sg/gov/moh/fhir/4.0.1/healtcert.json

```diff
@@ -0,0 +1,232 @@
+{
+  "resourceType": "Bundle",
+  "type": "collection",
+  "entry": [
+    {
+      "fullUrl": "urn:uuid:aaaa1321-4af5-424c-a0e1-ed3aab1c349d",
+      "resource": {
+        "resourceType": "Patient",
+        "extension": [
+          {
+            "url": "http://hl7.org/fhir/StructureDefinition/patient-nationality",
+            "extension": [
+              {
+                "url": "code",
+                "valueCodeableConcept": {
+                  "text": "SG"
+                }
+              }
+            ]
+          }
+        ],
+        "identifier": [
+          {
+            "type": {
+              "coding": [
+                {
+                  "system": "http://terminology.hl7.org/CodeSystem/v2-0203",
+                  "code": "PPN"
+                }
+              ]
+            },
+            "value": "E7831177G"
+          },
+          {
+            "type": {
+              "coding": [
+                {
+                  "code": "NRIC"
+                }
+              ]
+            },
+            "value": "S9098989Z"
+          }
+        ],
+        "name": [
+          {
+            "text": "Tan Chen Chen"
+          }
+        ],
+        "gender": "female",
+        "birthDate": "1990-01-15"
+      }
+    },
+    {
+      "fullUrl": "urn:uuid:bbbb1321-4af5-424c-a0e1-ed3aab1c349d",
+      "resource": {
+        "resourceType": "Specimen",
+        "type": {
+          "coding": [
+            {
+              "system": "http://snomed.info/sct",
+              "code": "258500001",
+              "display": "Nasopharyngeal swab"
+            }
+          ]
+        },
+        "collection": {
+          "collectedDateTime": "2020-09-27T06:15:00Z"
+        }
+      }
+    },
+    {
+      "fullUrl": "urn:uuid:99991321-4af5-424c-a0e1-ed3aab1c349d",
+      "resource": {
+        "resourceType": "Organization",
+        "name": "MOH",
+        "type": [
+          {
+            "coding": [
+              {
+                "system": "http://terminology.hl7.org/CodeSystem/organization-type",
+                "code": "govt",
+                "display": "Government"
+              }
+            ]
+          }
+        ]
+      }
+    },
+    {
+      "fullUrl": "urn:uuid:11111321-4af5-424c-a0e1-ed3aab1c349d",
+      "resource": {
+        "resourceType": "Practitioner",
+        "name": [
+          {
+            "text": "Dr Michael Lim"
+          }
+        ],
+        "qualification": [
+          {
+            "identifier": [
+              {
+                "system": "http://example.org/UniversityIdentifier",
+                "value": "MCR 123214"
+              }
+            ],
+            "code": {
+              "coding": [
+                {
+                  "system": "http://terminology.hl7.org/CodeSystem/v2-0360/2.7",
```

Comment:
This link results in 404, is this the link you might have been referring to? https://www.hl7.org/fhir/v2/0360/2.7/index.html

Also they do not have a code for MBBS, only MD, but most clinic doctors in SG are MBBS. Since the schema doesn't enforce this section, would it be better to not include it?

Reply:
Thanks for catching this, I actually never checked, it's data provided by MOH. I can change accordingly.

I have no idea what MBSS and MD are :/

Reply:
Thanks!

MBBS = Bachelor of Medicine and Bachelor of Surgery and MD = Doctor of Medicine. 
Cause I see that the example is stating their educational qualifications and doctors in SG minimally have an MBBS in order to practice medicine. But not all have MDs, and FHIR doesn't seem to have a code for MBBS. So I was wondering if we should be including their educational qualifications at all? It also might be a bit tricky to get this info from the clinics/labs too.

Reply:
The schema requires the qualification indeed. If it does, it's because MOH initially made it required (the case for almost all data). However FHIR compatible version does not require it.

I don't mind getting rid of it if you convince MOH to do so :/

Reply:
Hi @Nebulis we spoke with Crystal from MOH, she mentioned that they only require the doctor's MCR number and not their educational credentials. Would you be able to check with your contact at MOH if it is something that can be removed from the schema?
Thanks!

Reply:
will take this separately

Reply:
see #12